scripts/tasks/lint-imports.js

```diff
@@ -254,49 +292,60 @@ function lintImports() {
   }
 
   /**
-   * @param {ImportErrors} importErrors
+   * @param {ImportErrorGroup} errorGroup
+   * @param {string} relativePath
+   * @param {string} importPath
+   * @param {string} [alternative]
    */
-  function reportFilePathErrors(importErrors) {
-    const { pathNotFile, pathRelative, pathDeep, pathReExported } = importErrors;
-
-    logErrors(
-      pathNotFile,
-      'import path(s) do not reference physical files. This can break AMD imports. Please ensure the following imports reference physical files:'
-    );
-    logErrors(
-      pathRelative,
-      'example files are using relative imports. For example portability, please ensure that the following imports are absolute:'
-    );
-    logErrors(
-      pathDeep,
-      'example files are using deep imports. ' +
-        'To promote best practices, please only import from root-level files (<package-name> or <package-name>/lib/<file>):'
-    );
-    logErrors(
-      pathReExported,
-      'example files are directly importing from packages that office-ui-fabric-react re-exports. ' +
-        'Please change the following imports to reference office-ui-fabric-react instead:'
-    );
-
-    return pathNotFile.count > 0 || pathRelative.count > 0 || pathDeep.count > 0;
+  function _addError(errorGroup, relativePath, importPath, alternative) {
+    errorGroup.count++;
+    errorGroup.matches[relativePath] = errorGroup.matches[relativePath] || [];
+    errorGroup.matches[relativePath].push({ importPath, alternative });
   }
 
   /**
-   * @param {ImportErrorGroup} errorGroup
-   * @param {string} message
+   * @param {ImportErrors} importErrors
    */
-  function logErrors(errorGroup, message) {
-    if (errorGroup.count) {
-      console.error(`${chalk.red('ERROR')}: ${errorGroup.count} ${message}`);
-      console.error('-------------------------------------');
-      for (const filePath in errorGroup.matches) {
-        const { importPath, alternative } = errorGroup.matches[filePath];
-        console.error(`  ${filePath}: ${chalk.inverse(importPath)}`);
-        if (alternative) {
-          console.error(`        (use instead: '${alternative}')`);
+  function reportFilePathErrors(importErrors) {
```

Comment:
Maybe we should have a link to a new wiki page that outlines what's good and bad... we have one on those lines:
https://github.com/OfficeDev/office-ui-fabric-react/wiki/Coding-Style

Reply:
Good point. I made an issue #10431 about this.